README.md

```diff
@@ -295,8 +295,50 @@ Crawler::create()
     ->startCrawling($url);
 ```
 
+### Example 4: Crawling across requests
```

Comment:
❤️

Reply:
Looks like you found it already :) Just added it last night and wanted to prove read it once more before replying. I'll send an update shortly.